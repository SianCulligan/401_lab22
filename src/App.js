  
import React from 'react';

import Header from './components/Header.js';
import Footer from './components/Footer.js';
import Counter from './components/Counter.js';

import "./styles/styles.scss";
import "./styles/header.scss"
import "./styles/footer.scss"
import "./styles/counter.scss"

//REMEMBER: npm test --updateSnapshot

class App extends React.Component {
    render() {
        return (
            <React.Fragment>
                <Header />
                <Counter />
                <Footer />
            </React.Fragment>
        );
    }
}

export default App;