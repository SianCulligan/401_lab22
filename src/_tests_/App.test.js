import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App.js';


describe ('Snapshot test for the full app', () => {
    it('matches the snapshot', () => {
        const currentHTML = renderer.create(<App/>).toJSON();
        console.log(currentHTML);
        expect(currentHTML).toMatchSnapshot();
    });
});