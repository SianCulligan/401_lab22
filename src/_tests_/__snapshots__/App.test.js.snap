// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for the full app matches the snapshot 1`] = `
Array [
  <header
    className="header"
  >
    <h1>
      Counters R Fun
    </h1>
  </header>,
  <section
    className="counter"
  >
    <span
      className="downclicker"
      onClick={[Function]}
    >
       
      -
       
    </span>
    <span
      className="count "
    >
      0
    </span>
    <span
      className="upclicker"
      onClick={[Function]}
    >
       
      +
       
    </span>
  </section>,
  <footer
    className="footer"
  >
    Â© 2018 Code Fellows
  </footer>,
]
`;
